src/components/Layouts/Primary/TopBar/Items/NavMenu/NavMenu.tsx

```diff
@@ -23,6 +25,16 @@ const useStyles = makeStyles((theme: Theme) => ({
       display: 'none',
     },
   },
+  expand: {
+    transform: 'rotate(0deg)',
```

Comment:
Feel free to add this arrow to the nav dropdown Robert made.

Since I can't test it I'm not sure what this menu looks like. Is it in the nav header? Or is it the side menu? Or something else?

Reply:
@OzzieOrca  Already made for the nav dropdown Robert made.

(Closed)
![image](https://user-images.githubusercontent.com/60177750/122096293-68d1a100-cddc-11eb-81e8-6731c7c2f542.png)

(opened)
![image](https://user-images.githubusercontent.com/60177750/122096371-7d159e00-cddc-11eb-8d0a-36ac706bea69.png)